lib/Partitioner/Partitioner.cpp

```diff
@@ -26,6 +26,15 @@
 using namespace glow;
 using llvm::isa;
 
+namespace {
+// Sorted the std::pair<DAGNode *, uint64_t> based on the second from min to
+// max.
+auto sortMinMemory = [](const std::pair<DAGNode *, uint64_t> &a,
```

Comment:
Why is it a closure? It can be just a usual function, or?

Reply:
No special reason, just followed some code sample :P  Now removed.